src/routes/LandingPage/EventsBanner/index.scss

```diff
@@ -0,0 +1,44 @@
+.evts-banner {
+    position: relative;
+    overflow: hidden;
+    background-color: #691BC0;
+}
+
+.evts-desc {
+    color: white;
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 27px;
+    line-height: 32px;
+    margin: 90px;
+    display: block;
+}
+
+.evts-div {
+    position: relative;
+    display: block;
+    flex-direction: row;
+    justify-content: space-around;
+    left: -220px;
+    flex-wrap: wrap;
+}
+
+.evts-single {
+    display: inline;
+    font-family: Poppins;
+    text-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+    font-weight: 501;
+    font-size: 50px;
```

Comment:
I think perhaps this font-size is perhaps a little big; I think something thats approximately `2rem` would look great.

We've defined `1 rem` to be `18px` so that would come to ~`36px` but I think keeping the font-size in rem is a good idea.

Article on ems and rems if you're unfamiliar:
https://www.digitalocean.com/community/tutorials/css-rem-vs-em-units